docs/pages/management/join-services-to-your-cluster/join-token.mdx

```diff
@@ -0,0 +1,328 @@
+
+```code
+# Generate a short-lived invite token for a new Teleport SSH Service instance:
+$ tctl tokens add --ttl=5m --type=node 
+The invite token: (=presets.tokens.first=)
+This token will expire in 5 minutes.
+
+Run this on the new node to join the cluster:
+
+> teleport start \
+   --roles=node \
+   --token=(=presets.tokens.first=) \
+   --ca-pin=(=presets.ca_pin=) \
+   --auth-server=192.0.2.0:3025
+
+Please note:
+
+  - This invitation token will expire in 5 minutes
+  - 192.0.2.0:3025 must be reachable from the new node
+```
+
+In this command, we assigned the token the `node` type, indicating that it will
+belong to an SSH Service instance. 
+
+Copy the token so you can use it later in this guide.
+
+<Details title="Supported token types">
+
+Here are all the values we support for `--type` flag when creating a join token:
+
+|Role|Teleport Service|
+|---|---|
+|`app`|Application Service|
+|`auth`|Auth Service|
+|`bot`|Machine ID|
+|`db`|Database Service|
+|`discovery`|Discovery Service|
+|`kube`|Kubernetes Service|
+|`node`|SSH Service|
+|`proxy`|Proxy Service|
+|`windowsdesktop`|Windows Desktop Service|
+
+</Details>
+
+Administrators can generate tokens as they are needed. A Teleport process can
+use a token multiple times until its time to live (TTL) expires, with the
+exception of tokens with the `bot` type, which are used by Machine ID.
+
+To list all of the tokens you have generated, run the following command:
+
+```code
+$ tctl tokens ls
+Token                            Type Labels Expiry Time (UTC)
+-------------------------------- ---- ------ --------------------------
+(=presets.tokens.first=) Node        30 Mar 23 18:15 UTC (2m8s)
+```
+
+<Details scope={["oss","enterprise"]} title="An insecure alternative: static tokens" scopeOnly={false} >
+
+<Notice type="Danger">
+Use short-lived tokens instead of long-lived static tokens.
+Static tokens are easier to steal, guess, and leak.
+</Notice>
+
+Static tokens are defined ahead of time by an administrator and stored in the
+Auth Service's config file:
+
+```yaml
+# Config section in `/etc/teleport.yaml` file for the Auth Service
+auth_service:
+    enabled: true
+    tokens:
+    # This static token allows new hosts to join the cluster as "proxy" or "node"
+    - "proxy,node:secret-token-value"
+    # A token can also be stored in a file. In this example the token for adding
+    # new Auth Service instances are stored in /path/to/tokenfile
+    - "auth:/path/to/tokenfile"
+```
+</Details>
+
+### Start your Teleport process with the invite token
+
+Execute the following command on the host running your new Teleport process to
+add it to a cluster. Assign <Var name="join-token" /> to the token you generated
+earlier and <Var name="proxy-address" /> to the host and web port of your
+Teleport Proxy Service or Teleport Enterprise Cloud tenant (e.g.,
+`teleport.example.com:443`):
+
+```code
+$ sudo teleport configure \
```

Comment:
Consider `teleport node configure` here too, as it's more specific to generating a single SSH Service config.

Reply:
I wanted to show the more general command since this guide is for all services you can join. I've added a `Notice` indicating that you can use `teleport node configure` for SSH Service instances